src/components/AccountDrawer/AuthenticatedHeader.tsx

```diff
@@ -40,6 +40,7 @@ const AuthenticatedHeaderWrapper = styled.div`
   display: flex;
   flex-direction: column;
   flex: 1;
+  overflow: auto;
```

Comment:
Can you confirm if the whole portfolio drawer is supposed to scroll or just everything under the Tokens/NFTs/Pools/Activity Headers?
![scrollPortfolio](https://github.com/Uniswap/interface/assets/11512321/19bf7f01-3a1d-4151-b5ee-6fffcd424381)

Reply:
I'm assuming so because on main right now the whole portfolio drawer scrolls.